### ISSUE TITLE
hkust mtl_transformer_sp.json result

### ISSUE THREAD

**Issue by chenguoguo** [athena-team/athena#22]
@Some-random what is your number on the updated mtl_transformer_sp.json? Seems like it was couple (2%?) percents worse than the original configuration before https://github.com/athena-team/athena/pull/19?

**Comment by Some-random** [comment]
WER with updated mtl_transformer_sp.json (train for a total of 8 epochs on 4 gpu and divide learning rate by 10 at 22000 steps) on HKUST is 24.09 without MPC and 23.77 with MPC. Did you get better result with original mtl_transformer_sp.json (train for a total of 13 epochs and divide learning rate by 10 at 140000 steps)?

**Comment by chenguoguo** [comment]
My run was worse, something might be wrong, I’ll run it again.

**Comment by tjadamlee** [comment]
@Some-random do you use the same language model generated by "tools/kenlm/build/bin/lmplz -o 4 < examples/asr/hkust/data/text.seg  > examples/asr/hkust/data/4gram.arpa" to get the results ?

**Comment by Some-random** [comment]
Actually the LM model I used to get 24.09 was a 5-gram arpa... Sorry I forgot to mention it...

**Comment by tjadamlee** [comment]
So, please update the command in run.sh to support this, Thanks @Some-random

**Comment by Some-random** [comment]
> So, please update the command in run.sh to support this, Thanks @Some-random

I've updated run.sh and configuration json. I'll execute the full run.sh again to make sure it will get listed results.

**Comment by tjadamlee** [comment]
@Some-random can we close this issue or not

**Comment by Some-random** [comment]
> @Some-random can we close this issue or not

I've executed the whole shell scripts and got the results as listed in README.md. I think it's OK to close this issue for now

**Comment by chenguoguo** [comment]
Great! I'm closing this issue.
